demo-app/client/src/tests/post/regitration.service.spec.ts

```diff
@@ -1,7 +1,7 @@
-import {HttpClientModule, HttpRequest} from '@angular/common/http';
-import {HttpClientTestingModule, HttpTestingController} from '@angular/common/http/testing';
-import {async, inject, TestBed} from '@angular/core/testing';
-import {RegistrationService} from '../../../generated/controllers/Registration';
+import { HttpClientModule, HttpRequest } from '@angular/common/http';
```

Comment:
You have not run the linter, as I asked you earlier via `npm run lint; cd demo-app/client/; npm run lint`. Please, do. Formatting changes should be done only if they improve things.